es/src/main/scala/monix/connect/es/package.scala

```diff
@@ -1,14 +1,15 @@
 package monix.connect
 
-import com.sksamuel.elastic4s.http._
+import com.sksamuel.elastic4s._
 import monix.eval.Task
+import monix.execution.internal.InternalApi
 
 package object es {
 
-  implicit val taskFunctor: Functor[Task] = new Functor[Task] {
+  @InternalApi private[es] implicit val taskFunctor: Functor[Task] = new Functor[Task] {
     override def map[A, B](fa: Task[A])(f: A => B): Task[B] = fa.map(f)
   }
 
-  implicit val taskExecutor: Executor[Task] = (client: HttpClient, request: ElasticRequest) =>
+  @InternalApi private[es] implicit val taskExecutor: Executor[Task] = (client: HttpClient, request: ElasticRequest) =>
```

Comment:
maybe we could just refer to the existing instances from `elastic4s`? https://github.com/sksamuel/elastic4s/blob/master/elastic4s-effect-monix/src/main/scala/com/sksamuel/elastic4s/monix/instances/TaskInstances.scala

Reply:
The version of `monix` in `elastic4s` dependencies maybe the old version. Will this cause incompatibilities and conflicts?

Reply:
ah, good point! let's leave it as is then :)